.devcontainer/devcontainer.json

```
@@ -0,0 +1,36 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/kubernetes-helm-minikube
+{
+	"name": "Google Cloud - Teraform",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+
+	"features": {
+		// "ghcr.io/devcontainers/features/docker-in-docker:2": {
+		// 	"enableNonRootDocker": "true",
+		// 	"moby": "true"
+		// },
+		// "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {
+		// 	"version": "latest",
+		// 	"helm": "latest",
+		// 	"minikube": "latest"
+		// },
+		"ghcr.io/devcontainers/features/terraform:1": {},
+		// "ghcr.io/bartventer/arch-devcontainer-features/gcloud-cli:1": {}
+	},
+	
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "bash scripts/install-dependencies.sh"
```

I had to adjust the command like this on my side:
```suggestion
	"postCreateCommand": "bash .devcontainer/scripts/install-dependencies.sh"
```

---

That makes sense indeed, I wrote the scripts while running into dependency issues. I thought I rebuild the container afterwards, but apparently not.